integration_test/common/testdata/integration-metadata_oneof_field_expected-metric/input.yaml

```diff
@@ -0,0 +1,112 @@
+public_url: "https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/third-party/activemq"
```

Comment:
Which of these fields causes the golden error? Should we reduce the size of this test input, just like we did in #669, and add a comment on the line that should trigger the error?

Reply:
Changed in: [59870d7](https://github.com/GoogleCloudPlatform/ops-agent/pull/673/commits/59870d7c770575b84c837fcb1d7e1c705dd5405d)